modules/bcf/app/representers/bcf/api/v2_1/project_extensions/definitions.rb

```diff
@@ -37,29 +37,34 @@ def initialize(project:, user:)
       end
```

Comment:
This file can be deleted now I think

Reply:
You are right of course @oliverguenther. I would have sweared to have deleted it but now it actually is.